statistics/table.go

```diff
@@ -112,21 +112,51 @@ type HistColl struct {
 	Pseudo         bool
 }
 
+// TableMemoryUsage records tbl memory usage
+type TableMemoryUsage struct {
+	TableID         int64
+	TotalMemUsage   int64
+	ColumnsMemUsage map[int64]*ColumnMemUsage
+	IndicesMemUsage map[int64]*IndexMemUsage
+}
+
+// ColumnMemUsage records column memory usage
+type ColumnMemUsage struct {
+	TableID           int64
+	ColumnID          int64
+	HistogramMemUsage int64
+	CMSketchMemUsage  int64
+	FMSketchMemUsage  int64
```

Comment:
The memory usage tracking of `TopN` in `Column` and that of `TopN` and `FMSketch` in `Index` seem missing. We can add it in the PR or another new PR.

Reply:
The memUsage counting of `TopN` and `FMSketch` are not implemented now, I tended to implement them later.